packages/cypress/src/support/custom-assertions.ts

```diff
@@ -14,6 +14,23 @@ declare global {
   }
 }
 
+chai.use((chaiObj, utils) => {
+  function assertIsInViewport() {
```

Comment:
```suggestion
/**
   * Assert if the element chained from a command is in the viewport
   * @example
   * ```
   * cy.get('h1').should('be.inViewport');
   * cy.get('h1').should('not.be.inViewport');
   * ```
   * Adapted from https://www.webtips.dev/webtips/cypress/check-if-element-is-in-viewport
   */
  function assertIsInViewport() {
``````suggestion
  function assertIsInViewport() {
```